examples/mgui_in_dialog.py

```diff
@@ -0,0 +1,72 @@
+"""
+Method to get napari style in magicgui based windows
```

Comment:
I would maybe rename this example to something more representative like "inherit_viewer_style.py"?

Reply:
changed